packages/core/src/codewhisperer/nextEditPrediction/activation.ts

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import { PredictionTracker } from './PredictionTracker'
+import { PredictionKeyStrokeHandler } from './PredictionKeyStrokeHandler'
+import { getLogger } from '../../shared/logger/logger'
+import { ExtContext } from '../../shared/extensions'
+
+export let predictionTracker: PredictionTracker | undefined
+let keyStrokeHandler: PredictionKeyStrokeHandler | undefined
+
+export function activateEditTracking(context: ExtContext): void {
```

Comment:
As called out above

Can you please add a global try-catch + error log for all public methods which will be called for completions code path.